test/library/options-test.ts

```diff
@@ -29,6 +29,20 @@ describe('options test', () => {
 		async resetKey(_key: string): Promise<void> {}
 	}
 
+	it('should allow an empty options object', async () => {
+		expect(rateLimit()).not.toThrow()
```

Comment:
Oh, this test is wrong. It should be something like this:
```suggestion
		expect(rateLimit).not.toThrow()
```

or possibly

```suggestion
		expect(() => rateLimit()).not.toThrow()
```

Otherwise expect is trying to execute the returned middleware function with no arguments to see if *it* throws, which is not what the test is supposed to be testing.